Softeq.XToolkit.Common.Tests/Extensions/DateTimeExtensionsTests/DateTimeExtensionsTests.cs

```diff
@@ -62,5 +80,78 @@ public void IsYesterday_ForDateTime_ReturnsExpectedValue(DateTime testDate, bool
 
             Assert.Equal(expectedIsYesterday, isYesterday);
         }
+
+        [Fact]
+        public void IsInCurrentWeek_ForPreviousAndNextDaysOfToday_ReturnsTrueCorrectAmountOfTimes()
+        {
+            var weekLength = 7;
+
+            var date = DateTime.Today;
+            var isInCurrentWeek = date.IsInCurrentWeek();
+            var count = isInCurrentWeek ? 1 : 0;
+
+            for (var i = 1; isInCurrentWeek; i++)
```

Comment:
complex dynamic test, can be better to use hardcoded week for check boundaries

Reply:
I don't understand how I can use hardcoded week to check if it is current week, the method itself is dynamic. I can rework the method itself to provide week as a parameter. Will this be ok with you or did I miss something?

Reply:
Actually I can just remove this test, I guess it's covered by other tests